idaes/generic_models/unit_models/heat_exchanger_lc.py

```diff
@@ -0,0 +1,203 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+Extension of the IDAES heat exchanger model for transient simulations
+"""
+
+__author__ = "Rusty Gentile, John Eslick, Andrew Lee"
+
+from pyomo.environ import Var, Param
+from pyomo.dae import DerivativeVar
+from idaes.core import declare_process_block_class
+from .heat_exchanger import HeatExchangerData
+
+
+@declare_process_block_class("HeatExchangerLumpedCapacitance",
+                             doc="0D heat exchanger for transient simulations")
+class HeatExchangerLumpedCapacitanceData(HeatExchangerData):
+
+    CONFIG = HeatExchangerData.CONFIG(implicit=True)

```

Comment:
Unless you want to add configuration options, I don't think you need this line since CONFIG should already exist as part of the inherited HeatExchangerData.

Reply:
Removed with my last commit. Thanks for the feedback!